Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```diff
@@ -37,7 +37,7 @@
       ballistic-ammo: !type:Container
         ents: []
   - type: StaticPrice
-    price: 500
+    price: 720
```

Comment:
Snipers are already undertuned and even with the planned zoom probably will be so not sure why the price should go up.

Reply:
The price is increased because a sniper rifle can't cost the same as a regular gun. Most of the change in weapon prices is an average between the RP and the cost of the weapon at Cargo. It may be useless, but it is still a sniper rifle.